src/components/Images/Images.js

```diff
@@ -198,10 +172,12 @@ export const CloudinaryImage = ({
 
     return (
       <img
-        data-src={svgUrl}
+        key={publicId}
+        src={svgUrl}
         className={[styles.Svg, ...imageClasses].join(' ')}
         alt={alt}
         fetchpriority={fetchpriority}
+        loading="lazy"
```

Comment:
natively supported by all browsers

Reply:
Same here, let's respect the prop.

Reply:
gotta give props where props are due, respect.